### DISCUSSION TITLE
SEO: Ciência ou Charlatanismo?

### DISCUSSION THREAD

**Discussion by vitor-mendrone-deel** [frontendbr/forum#88] [Perguntas e Respostas]
De cara peço perdão pelo título 'mamilico', porém gostaria de saber mais sobre o posicionamento entre os desenvolvedores sobre o assunto. 

Ouvi de alguém a alguns anos (acho que foi no primeiro FEMUG que fui) que SEO era feng shui de programador: Coloca essa cama virada pro sol que terá mais sorte na vida ou, no nosso caso, coloque essa palavrinha na URL e seu site decola pro topo das buscas.
- Quanto tempo dedicam ao assunto? (Tempo de estudo, testes, etc)
- Vocês tem experiências reais de trabalho em "upar" o ranqueamento de um site?
- As empresas que trabalham tem profissionais específicos da área ou a responsa é dos devs?

**Comment by bernardodiasc** [comment]
hehehe considerando que deixar a cama bem posicionada em relação ao sol ajuda a evitar mofo, elimina ácaros, mal odores e etc, possibilitando boas noites de sono e melhor qualidade de vida, o que por sua vez deixa o estado de espirito mais positivo e mais aberto para as coisas boas que a vida oferece... não chamaria isso de charlatanismo, são coisas sutis com resultados as vezes indiretos. Assim como SEO, 1 palavrinha pode, de fato, ser decisiva no resultado nas buscas.

**Comment by vitor-mendrone-deel** [comment]
@bernardodiasc HAHAHAHAH excelente analogia!

Essa é a grande complexidade do SEO, as vezes tudo é sutil e incerto demais.

**Comment by paulovalente** [comment]
Cara posso dizer que vale muito apena.
Este é meu site:  https://www.freelancerfrontend.com.br/
procure por: Recorte de PSD para HTML.
Utilizo em 1 h1. E posso dizer, que tudo vai depender do seu linkbuilding.
Outro exemplo que melhorou bem: SSL :D
Olha um exemplo que coloquei: http://puu.sh/nHDc7/6fcb9cb84d.png
Olha como subiu a parada! Curto demais SEO!! e no brasil é muito precario!

**Comment by clarabatt** [comment]
Tenho estudado e tem muitas coisas que ajudam a impulsionar, como sitemaps, link building, o estudo das palavras e como o seu público se comporta na hora de pesquisar. 

Em um projeto que fiz realizei uma pesquisa com algumas pessoas e pedi pra elas 'buscarem' no google pelo produto que a empresa oferecia. Me impressionei com os resultados, os usuários comuns são mais criativos que nós na hora de pesquisar kkkkkkk tinham palavras que eu nem imaginaria e foi muito produtivo.

**Comment by felipefialho** [comment]
Acredito que a gente necessita saber o suficiente, e é possível aprender até mesmo com os sites pessoais (tentando subir o rank dele).

Mas no meu caso especificamente, faço o básico:
- Dentro do possível, tabalhar bem com acessibilidade 
- Estrutura mais semântica possível
- Uso de `roles` básicas
- schema.org

Parece que o Google inclusive alterou algumas coisas no rank, favorecendo sites com foco em mobile, portando um código bem construído se faz ainda mais necessário.

**Comment by felipeuntill** [comment]
### TL:DR

O SEO _(masculino porque é um processo)_ passou por diversas adaptações, apesar do bing e outros mecanismos de busca terem ganho força expressiva nos últimos anos o [Market Share](https://www.netmarketshare.com/search-engine-market-share.aspx?qprid=4&qpcustomd=0) do Google continua devastador e o foco das otimizações em geral é quase que restrito a ele.

Se formos dividir a evolução do processo de crawling dos robôs de busca do Google em períodos teríamos as seguintes fases:

### Resultado baseado no conteúdo das páginas (Content Crawling)

Inicialmente os algoritmos de busca percorriam as páginas da internet e registravam as palavras e suas ocorrências, resumindo, se você escreve-se um artigo em seu blog e em cada parágrafo  colocasse a frase _"NodeJs"_ você estaria aumentando a relevância  do seu site para este termo (Keyword stuffing).
Com o passar dos anos a comunidade começou a utilizar este e outros fatores de ranqueamento baseados no conteúdo das páginas (Title, Meta Description, Meta Keyworks e HTML Semântico) para otimizar os resultados das consultas de forma tendenciosa.

Inicialmente este tipo de técnica acabou por ajudar a disseminar os requisitos de qualidade da W3C a adesão ao Tableless dentre outros fatores. contudo os resultados das buscas começaram a ser forjados e muitas vezes pouco expressivos, páginas otimizadas porém conteúdo irrelevante passaram a ganhar destaque sobre as não otimizadas que apresentavam conteúdo relevante, até mesmo a wikipedia perdeu sua primeira posição em inúmeros resultados de busca. 

Os engenheiros do Google entenderam que um site não deveria conseguir se posicionar ante a concorrência por contra própria eles precisavam saber se o conteúdo realmente era bom.

### Resultado baseado em autoridade (Page Rank)

Neste período o mercado de otimização para buscas orgânicas ja estava maduro, surgiram agências especializadas, serviços facilitadores (SEO Moz, Majestic SEO, dentre outros) e o SEO que outrora era perfumaria passou a ser um requisito. 

Para resolver o problema das páginas garantirem vantagem sobre as demais apenas manipulando seu conteúdo, o Google modificou seu algoritmo para considerar a autoridade dos sites.

O algoritmo de ranqueamento baseado em autoridade basicamente identificava a quantidade de sites da web que tinham um link para o seu site, e qual era o conteúdo da ancora, resumindo, se vários sites da web apontassem para o seu site com links contendo a palavra pesquisada, o google entendia você era uma referência naquele assunto e que portanto era o mais indicado para aparecer nos resultados. Não obstante, o algoritmo também considerava que páginas quais eram linkadas em sites de grande relevância também apresentavam conteúdo de qualidade. 

Exemplo de links que fariam uma página ser autoridade para a palavra _"NodeJS"_

``` html
<a href="www.seusite.com">Artigos Sobre NodeJS</a>
<a href="www.seusite.com">Tutoriais de NodeJS</a>
<a href="www.seusite.com">NodeJs para iniciantes</a>
```

Com o passar do tempo as campanhas de otimização focadas em conteúdo perderam espaço e o mercado procurou outras formas de burlar o algoritmo o que culminou na criação do mercado de compra de links e artigos, criação de sites fictícios unicamente para se criar links para os sites otimizados (**Link Farm**), dentre outras formas de trocas de links de forma não natural, estes artifícios ficaram conhecidos pelo termo **Link Building**.

### Resultado baseado na experiência do usuário (User  Experience)

Com o passar dos anos o Google entendeu que revelar de forma direta seus fatores de ranqueamento servia como fonte de engajamento para a engenharia reversa portanto hoje a empresa não declara de forma direta as premissas de posicionamento do algoritmo porém na grande maioria das vezes em que um engenheiro da empresa fala sobre a qualidade dos resultados eles apontam para as seguintes premissas:
- Mobilidade (sites responsivos ou com redirect para um site focado na experiência mobile)
- Engajamento social (quão comentado um site é nas redes sociais)
- Tempo de carregamento (quão rápido seu site carrega)

dentre outras premissas quais são focadas na experiência do usuário, claro que todas os meios de otimização considerados nas versões anteriores do não foram totalmente desconsiderados eles continuam presentes mas perderam peso.

---

### Respondendo suas Perguntas:

_Quanto tempo dedicam ao assunto? (Tempo de estudo, testes, etc)_
**R**: Fui um early adopter do SEO na época eu trabalhava como front-end em agências da capital de São Paulo e utilizei este recurso pra se promover profissionalmente. 
Até o início do ano passado eu possuía 2 clientes de otimização(Freelances) quais optei por desistir de atender visto que justificar resultados estava cada vez mais difícil e trabalhoso, no mesmo período parei de estudar e depreender testes visto que criar site e conteúdo para testar exigia muito tempo)

Vocês tem experiências reais de trabalho em "upar" o ranqueamento de um site?
**R**: No inicio era fácil e simples manipular o algoritmo portanto consegui muitos resultados expressivos, aumentos de até 500% em visitação em um ano, contudo nos últimos anos de trabalho estava difícil justificar qualquer resultado.

As empresas que trabalham tem profissionais específicos da área ou a responsa é dos devs?
**R**: Depende do tamanho da empresa, normalmente em agências existe pessoal dedicado mas na maioria dos casos não são desenvolvedores e as campanhas são mais focadas em desenvolvimento de conteúdo que em manipulação técnica.

---

### Agora um espacinho para minha opinião pessoal:

Sou do tipo que não para em emprego mas só posso falar sobre o mercado em SP.

Tirando algumas pequenas agências e alguns desavisados o mercado tem ignorado o SEO, os resultados são inexpressivos e incertos o que dificulta na venda e entrega.

O processo de otimização para mecanismos de busca é um cancer na qualidade das buscas, se um site é tão bom em determinado assunto a ponto de precisar **forçar** sua relevância ele certamente não é tão bom quanto imagina.

O Google entendeu que se você clica em uma página e volta para o resultado de busca procurando outra, a página anterior certamente não foi o melhor resultado pra você portanto não adianta forçar um engajamento pois se a página tem qualidade ela cria sua própria relevância.

> Considere seu site como uma loja em um shopping não adianta você encher sua vitrine com letreiros de promoções saldões e etc pois quando o cliente entrar ele não encontrar o que realmente procure ele certamente vai sair e procurar em outra.

**Comment by vitorkobbaz** [comment]
Abri uma agência há 2 anos atrás e hoje o meu cliente mais antigo obtém excelentes resultados com SEO (Aproximadamente 20 mil visitas orgânicas por mês), mas a dificuldade é que demora para dar resultado, você tem que se dedicar muito a construção de backlinks de qualidade e muitos desistem no meio do caminho. Por isso a melhor forma é se conscientizar que é um trabalho de formiguinha e de longo prazo, com muito monitoramento e geração de conteúdo de qualidade. A vantagem é que as visitas conquistadas mantém-se sólidas por muito tempo e você não precisa ficar refém dos investimentos em Ads.

**Comment by marconez777** [comment]
A melhor maneira de posicionar um site com ctz é com uma estratégia chamada PBN!
antigamente era muito dificil fazer uma rede de pbn no Brasil devido a  dificuldade de encontrar dominios expirados brasileiros com métricas boas. Mas com essa ferramenta https://dominiosexpiradosbr.com.br/ tudo mudou

**Comment by wilsonneto-swe** [comment]
PBN?? 
A melhor maneira de fazer SEO é conteúdo de qualidade, o google mede quanto tempo a galera passa no seu site, se a galera só entra e sai, não adianta nenhum PBN, Link Building ou artimanhas milaborantes... 

Se os usuários clicam e ficam em seu site, show, essa é a chave, a experiência do usuário e conteúdos.

**Comment by marconez777** [comment]
E quem disse que PBN não pode ter conteúdo de qualidade e trafego real??

Só porque se trada de uma rede de blogs privada, não significa que deve ser feita apenas com o intuito de colocar o link para o site. Ai que está o erro de todos que fazem PBN, fazem mal feito!

PBN deve ser feita por web designer, alocadas em hospedagens de boa procedência, com conteúdo de qualidade produzido por redatores. Deve-se trabalhar  palavras chaves de baixa concorrência no site de pbn para que ele posicione e gere trafego.  

Tenho vários projetos e clientes bem posicionados no Google, Tranbalho com SEO a mais de 4 anos e posso falar com propriedade, PBN é a melhor e mais efetiva estratégia de Linkbuilding. 

**SE FOR FEITA COM QUALIDADE**

**Comment by felquis** [comment]
Conteúdo é Rei
